2016-01-17 18:59:57,559 (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
2016-01-17 18:59:57,560 (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
2016-01-17 18:59:57,560 (0.000) SELECT `mysite_question`.`id`, `mysite_question`.`text`, `mysite_question`.`pollcnt`, `mysite_question`.`st`, `mysite_question`.`dt`, `mysite_question`.`commitcnt` FROM `mysite_question` WHERE (`mysite_question`.`st` <= '2016-01-17 10:59:57.546701' AND `mysite_question`.`dt` > '2016-01-17 10:59:57.546701'); args=(u'2016-01-17 10:59:57.546701', u'2016-01-17 10:59:57.546701')
2016-01-17 19:00:00,081 管理员 进入 管理页面
2016-01-17 19:00:01,779 (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
2016-01-17 19:00:01,779 (0.000) SELECT COUNT('*') AS `__count` FROM `mysite_user`; args=('*',)
2016-01-17 19:00:01,780 (0.000) SELECT cache_key, value, expires FROM `my_cache_table` WHERE cache_key = ':1:django.contrib.sessions.cached_dbnforcapxjifzuzrwoqttlibpq7968t8r'; args=[u':1:django.contrib.sessions.cached_dbnforcapxjifzuzrwoqttlibpq7968t8r']
2016-01-17 19:00:01,788 (0.000) SELECT `mysite_user`.`id`, `mysite_user`.`username`, `mysite_user`.`idsn`, `mysite_user`.`addr`, `mysite_user`.`birth`, `mysite_user`.`sex`, `mysite_user`.`phone`, `mysite_user`.`pwd`, `mysite_user`.`nation`, `mysite_user`.`hashsn` FROM `mysite_user` LIMIT 1; args=()
2016-01-17 19:00:03,799 管理员 进入 管理页面
2016-01-17 19:00:53,005 管理员 进入 管理页面
2016-01-17 19:01:05,080 管理员 进入 管理页面
2016-01-17 19:18:15,178 管理员 进入 管理页面
2016-01-17 19:23:51,294 管理员 进入 管理页面
2016-01-17 19:23:52,621 管理员 进入 管理页面
2016-01-17 19:24:40,304 管理员 进入 管理页面
2016-01-17 19:24:48,548 管理员 进入 管理页面
2016-01-17 19:24:55,997 管理员 进入 管理页面
2016-01-17 19:25:17,626 管理员 进入 管理页面
2016-01-17 19:25:22,294 管理员 进入 管理页面
2016-01-17 19:26:33,519 管理员 进入 管理页面
2016-01-17 19:31:55,513 管理员 进入 管理页面
2016-01-17 19:32:41,664 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-17 19:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-17 19:32:41,677 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-17 20:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-17 19:33:27,720 管理员 进入 管理页面
2016-01-17 19:41:38,396 管理员 进入 管理页面
2016-01-17 19:43:01,480 管理员 进入 管理页面
2016-01-17 19:44:55,669 管理员 进入 管理页面
2016-01-17 19:45:07,737 管理员 进入 管理页面
2016-01-17 19:46:02,703 管理员 进入 管理页面
2016-01-17 19:47:27,936 管理员 进入 管理页面
2016-01-17 20:04:46,087 管理员 进入 管理页面
2016-01-17 20:09:58,478 管理员 进入 管理页面
2016-01-17 20:10:01,167 管理员 进入 管理页面
2016-01-17 20:12:37,519 管理员 进入 管理页面
2016-01-17 20:13:02,226 管理员 进入 管理页面
2016-01-17 20:13:04,757 管理员 进入 管理页面
2016-01-17 20:13:59,949 管理员 进入 管理页面
2016-01-17 20:15:01,499 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-17 20:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-17 20:15:01,511 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-17 23:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-17 20:15:10,353 管理员 进入 管理页面
2016-01-17 20:26:08,933 管理员 进入 管理页面
2016-01-17 20:26:19,384 管理员 进入 管理页面
2016-01-17 20:56:22,825 管理员 进入 管理页面
2016-01-17 21:00:06,163 /Library/Python/2.7/site-packages/django/forms/fields.py:885: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  if value == k or text_value == force_text(k):

2016-01-17 21:02:17,849 管理员 进入 管理页面
2016-01-17 21:24:16,518 管理员 进入 管理页面
2016-01-17 21:24:37,814 管理员 进入 管理页面
2016-01-17 21:24:56,329 /Library/Python/2.7/site-packages/django/forms/fields.py:885: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  if value == k or text_value == force_text(k):

2016-01-17 21:25:51,970 管理员 进入 管理页面
2016-01-17 21:28:12,088 管理员 进入 管理页面
2016-01-17 21:35:20,239 管理员 进入 管理页面
2016-01-17 21:36:53,811 管理员 进入 管理页面
2016-01-17 21:42:47,988 管理员 进入 管理页面
2016-01-17 21:42:50,317 管理员 进入 管理页面
2016-01-17 22:04:18,753 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-17 00:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-17 22:04:18,763 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-17 23:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-17 22:05:09,502 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-17 00:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-17 22:05:09,513 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-17 23:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-17 22:05:46,406 管理员 进入 管理页面
2016-01-17 22:05:50,352 管理员 进入 管理页面
2016-01-17 22:05:52,646 管理员 进入 管理页面
2016-01-17 22:06:34,604 管理员 进入 管理页面
2016-01-17 22:07:46,835 /Library/Python/2.7/site-packages/django/forms/fields.py:885: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  if value == k or text_value == force_text(k):

2016-01-17 22:08:35,697 /Library/Python/2.7/site-packages/django/forms/fields.py:885: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  if value == k or text_value == force_text(k):

2016-01-17 22:10:38,571 /Library/Python/2.7/site-packages/django/forms/fields.py:885: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  if value == k or text_value == force_text(k):

2016-01-17 22:10:41,965 管理员 进入 管理页面
2016-01-17 22:16:44,599 管理员 进入 管理页面
2016-01-17 22:17:18,853 管理员 进入 管理页面
2016-01-17 22:19:56,945 管理员 进入 管理页面
2016-01-17 22:20:06,268 管理员 进入 管理页面
2016-01-17 22:20:17,603 管理员 进入 管理页面
2016-01-17 22:20:45,977 管理员 进入 管理页面
2016-01-17 22:20:57,574 管理员 进入 管理页面
2016-01-17 22:21:11,605 /Library/Python/2.7/site-packages/django/forms/fields.py:885: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  if value == k or text_value == force_text(k):

2016-01-17 22:21:13,096 管理员 进入 管理页面
2016-01-17 22:21:15,394 管理员 进入 管理页面
2016-01-17 22:23:01,887 管理员 进入 管理页面
2016-01-17 22:23:06,060 管理员 进入 管理页面
2016-01-17 22:26:04,292 管理员 进入 管理页面
2016-01-17 22:39:28,241 管理员 进入 管理页面
2016-01-17 22:39:36,661 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-17 00:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-17 22:39:36,674 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-17 23:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-17 22:40:02,329 管理员 进入 管理页面
2016-01-17 22:40:06,109 管理员 进入 管理页面
2016-01-17 22:40:14,121 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-17 00:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-17 22:40:14,132 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-17 23:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-17 22:40:39,045 管理员 进入 管理页面
2016-01-17 22:45:25,921 管理员 进入 管理页面
2016-01-17 23:01:53,210 管理员 进入 管理页面
2016-01-17 23:02:20,656 管理员 进入 管理页面
2016-01-17 23:02:34,107 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-17 00:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-17 23:02:34,119 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-17 23:59:00) while time zone support is active.
  RuntimeWarning)

2016-01-18 00:00:01,750 管理员 进入 管理页面
2016-01-18 00:00:11,258 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-18 00:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-18 00:00:11,270 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-18 22:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-18 02:31:32,759 管理员 进入 管理页面
2016-01-18 02:32:22,396 管理员 进入 管理页面
2016-01-18 02:32:39,488 管理员 进入 管理页面
2016-01-18 02:32:57,020 管理员 进入 管理页面
2016-01-18 02:35:01,620 管理员 进入 管理页面
2016-01-18 02:35:20,935 管理员 进入 管理页面
2016-01-18 02:37:20,176 管理员 进入 管理页面
2016-01-18 02:37:28,204 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-18 00:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-18 02:37:28,215 /Library/Python/2.7/site-packages/django/db/models/fields/__init__.py:1474: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-18 23:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-18 02:38:44,392 管理员 进入 管理页面
2016-01-23 15:15:12,563 管理员 进入 管理页面
2016-01-23 15:18:32,646 管理员 进入 管理页面
2016-01-23 15:44:35,536 管理员 进入 管理页面
2016-01-23 15:54:56,370 管理员 进入 管理页面
2016-01-23 15:54:57,907 管理员 进入 管理页面
2016-01-23 16:11:32,121 管理员 进入 管理页面
2016-01-23 16:12:32,701 管理员 进入 管理页面
2016-01-23 16:12:49,860 管理员 进入 管理页面
2016-01-23 16:14:12,431 管理员 进入 管理页面
2016-01-23 16:14:53,109 管理员 进入 管理页面
2016-01-23 16:15:21,644 管理员 进入 管理页面
2016-01-23 16:19:26,387 管理员 进入 管理页面
2016-01-23 16:22:57,884 管理员 进入 管理页面
2016-01-23 16:23:00,476 管理员 进入 管理页面
2016-01-23 16:23:20,712 管理员 进入 管理页面
2016-01-23 16:23:32,838 管理员 进入 管理页面
2016-01-23 16:23:38,726 管理员 进入 管理页面
2016-01-23 16:23:49,724 管理员 进入 管理页面
2016-01-23 16:24:38,023 管理员 进入 管理页面
2016-01-23 16:50:26,413 管理员 进入 管理页面
2016-01-23 17:14:43,968 管理员 进入 管理页面
2016-01-23 17:16:24,858 管理员 进入 管理页面
2016-01-23 17:18:28,200 管理员 进入 管理页面
2016-01-23 17:24:22,184 管理员 进入 管理页面
2016-01-23 17:25:11,085 管理员 进入 管理页面
2016-01-23 17:27:47,076 管理员 进入 管理页面
2016-01-23 17:48:35,266 管理员 进入 管理页面
2016-01-23 20:47:02,490 管理员 进入 管理页面
2016-01-23 20:47:05,787 管理员 进入 管理页面
2016-01-23 20:47:08,831 管理员 进入 管理页面
2016-01-23 20:47:25,005 管理员 进入 管理页面
2016-01-23 20:47:58,356 管理员 进入 管理页面
2016-01-23 20:48:27,598 管理员 进入 管理页面
2016-01-23 20:51:45,005 管理员 进入 管理页面
2016-01-23 20:52:40,615 管理员 进入 管理页面
2016-01-23 20:53:01,911 管理员 进入 管理页面
2016-01-23 20:53:32,541 管理员 进入 管理页面
2016-01-23 20:56:17,333 管理员 进入 管理页面
2016-01-23 23:25:34,947 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-23 23:25:34,948 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-23 23:25:59,504 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-23 23:25:59,504 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-23 23:26:15,962 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-23 23:26:15,968 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-23 23:26:52,798 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-23 23:26:53,267 Not Found: /favicon.ico
2016-01-23 23:26:54,381 管理员 进入 管理页面
2016-01-23 23:26:55,726 /usr/local/lib/python2.7/dist-packages/bootstrapform/templatetags/bootstrap.py:80: RemovedInDjango110Warning: render() must be called with a dict, not a Context.
  return template.render(context)

2016-01-23 23:26:57,541 Internal Server Error: /settext/1
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/root/poll/mysite/views.py", line 715, in settext
    form = SetTextForm(initial = {"content":Text.objects.all().filter(id = id)[0].content})
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 297, in __getitem__
    return list(qs)[0]
IndexError: list index out of range
2016-01-23 23:27:12,205 管理员 进入 管理页面
2016-01-23 23:28:22,955 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-23 23:28:22,960 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-23 23:28:25,633 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-23 23:28:36,843 Not Found: /asdfsf
2016-01-23 23:29:02,144 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-23 23:29:02,144 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-23 23:29:18,767 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-23 23:29:43,289 Not Found: /http:/zc.qq.com/cgi-bin/common/attr
2016-01-23 23:31:12,037 Not Found: /favicon.ico
2016-01-23 23:31:16,213 Not Found: /getseeed
2016-01-23 23:34:06,420 Not Found: /favicon.ico
2016-01-23 23:34:50,598 Internal Server Error: /auth/10000/70e4b9d5538ad307c6210bef07a50117
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 235, in get_response
    response = middleware_method(request, response)
  File "/usr/local/lib/python2.7/dist-packages/django/contrib/sessions/middleware.py", line 50, in process_response
    request.session.save()
  File "/usr/local/lib/python2.7/dist-packages/django/contrib/sessions/backends/cached_db.py", line 65, in save
    super(SessionStore, self).save(must_create)
  File "/usr/local/lib/python2.7/dist-packages/django/contrib/sessions/backends/db.py", line 80, in save
    return self.create()
  File "/usr/local/lib/python2.7/dist-packages/django/contrib/sessions/backends/db.py", line 49, in create
    self._session_key = self._get_new_session_key()
  File "/usr/local/lib/python2.7/dist-packages/django/contrib/sessions/backends/base.py", line 155, in _get_new_session_key
    if not self.exists(session_key):
  File "/usr/local/lib/python2.7/dist-packages/django/contrib/sessions/backends/cached_db.py", line 60, in exists
    if session_key and (self.cache_key_prefix + session_key) in self._cache:
  File "/usr/local/lib/python2.7/dist-packages/django/core/cache/backends/base.py", line 203, in __contains__
    return self.has_key(key)
  File "/usr/local/lib/python2.7/dist-packages/django/core/cache/backends/db.py", line 180, in has_key
    [key, connection.ops.adapt_datetimefield_value(now)])
  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python2.7/dist-packages/django/db/utils.py", line 95, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/mysql/base.py", line 112, in execute
    return self.cursor.execute(query, args)
  File "/usr/lib/python2.7/dist-packages/MySQLdb/cursors.py", line 174, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib/python2.7/dist-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1146, "Table 'poll.my_cache_table' doesn't exist")
2016-01-23 23:37:39,807 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-23 23:37:39,808 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-23 23:37:44,348 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-23 23:37:44,348 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-23 23:37:52,208 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-23 23:37:56,368 Internal Server Error: /init
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/root/poll/mysite/views.py", line 53, in init
    if DEBUG or Var.objects.all().filter("bind") == None:
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 790, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 808, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/sql/query.py", line 1243, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/sql/query.py", line 1269, in _add_q
    allow_joins=allow_joins, split_subq=split_subq,
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/sql/query.py", line 1146, in build_filter
    arg, value = filter_expr
ValueError: too many values to unpack
2016-01-23 23:38:24,880 Internal Server Error: /init
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/root/poll/mysite/views.py", line 53, in init
    if DEBUG or Var.objects.all().filter("bind") == None:
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 790, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 808, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/sql/query.py", line 1243, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/sql/query.py", line 1269, in _add_q
    allow_joins=allow_joins, split_subq=split_subq,
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/sql/query.py", line 1146, in build_filter
    arg, value = filter_expr
ValueError: too many values to unpack
2016-01-23 23:38:43,141 /usr/local/lib/python2.7/dist-packages/bootstrapform/templatetags/bootstrap.py:80: RemovedInDjango110Warning: render() must be called with a dict, not a Context.
  return template.render(context)

2016-01-23 23:39:29,139 Not Found: /favicon.ico
2016-01-23 23:40:19,494 管理员 进入 管理页面
2016-01-23 23:41:01,607 管理员 进入 管理页面
2016-01-23 23:42:25,081 管理员 进入 管理页面
2016-01-23 23:43:14,657 管理员 进入 管理页面
2016-01-23 23:48:41,328 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-23 23:48:41,329 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-23 23:48:41,588 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-23 23:48:45,462 管理员 进入 管理页面
2016-01-23 23:48:47,642 /usr/local/lib/python2.7/dist-packages/bootstrapform/templatetags/bootstrap.py:80: RemovedInDjango110Warning: render() must be called with a dict, not a Context.
  return template.render(context)

2016-01-23 23:49:13,129 Not Found: /favicon.ico
2016-01-23 23:50:12,090 管理员 进入 管理页面
2016-01-23 23:53:22,922 管理员 进入 管理页面
2016-01-23 23:54:54,428 管理员 进入 管理页面
2016-01-23 23:55:58,974 管理员 进入 管理页面
2016-01-23 23:56:23,764 管理员 进入 管理页面
2016-01-23 23:56:39,851 管理员 进入 管理页面
2016-01-23 23:57:19,603 /usr/local/lib/python2.7/dist-packages/django/forms/fields.py:849: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  if value == k or text_value == force_text(k):

2016-01-23 23:57:21,148 管理员 进入 管理页面
2016-01-23 23:58:12,380 管理员 进入 管理页面
2016-01-23 23:59:22,373 管理员 进入 管理页面
2016-01-24 00:02:20,104 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 00:02:20,105 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 00:02:26,630 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 00:02:31,171 管理员 进入 管理页面
2016-01-24 00:02:50,565 管理员 进入 管理页面
2016-01-24 00:02:54,564 /usr/local/lib/python2.7/dist-packages/bootstrapform/templatetags/bootstrap.py:80: RemovedInDjango110Warning: render() must be called with a dict, not a Context.
  return template.render(context)

2016-01-24 00:03:19,628 管理员 进入 管理页面
2016-01-24 00:03:41,614 /usr/local/lib/python2.7/dist-packages/django/forms/fields.py:849: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  if value == k or text_value == force_text(k):

2016-01-24 00:03:43,122 管理员 进入 管理页面
2016-01-24 00:05:30,300 Not Found: /favicon.ico
2016-01-24 00:06:22,813 管理员 进入 管理页面
2016-01-24 00:08:22,426 管理员 进入 管理页面
2016-01-24 00:09:16,306 管理员 进入 管理页面
2016-01-24 00:10:25,077 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-24 00:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 00:10:25,078 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-24 00:15:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 00:10:46,821 Internal Server Error: /auth/10005/412295b892e75f8abe93eabcc9ed411d
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 158, in get_response
    % (callback.__module__, view_name))
ValueError: The view mysite.views.auth didn't return an HttpResponse object. It returned None instead.
2016-01-24 00:10:47,064 Not Found: /favicon.ico
2016-01-24 00:11:16,531 Internal Server Error: /auth/10006/c35d682d742cabe64892d9ba67f6a045
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 158, in get_response
    % (callback.__module__, view_name))
ValueError: The view mysite.views.auth didn't return an HttpResponse object. It returned None instead.
2016-01-24 00:11:16,774 Not Found: /favicon.ico
2016-01-24 00:12:09,641 Internal Server Error: /auth/10007/6b438fd9b3205459905ee30f3929fa59
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 158, in get_response
    % (callback.__module__, view_name))
ValueError: The view mysite.views.auth didn't return an HttpResponse object. It returned None instead.
2016-01-24 00:12:09,879 Not Found: /favicon.ico
2016-01-24 00:14:18,617 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 00:14:18,620 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 00:14:25,403 Internal Server Error: /auth/10008/133c1ce01336c359e42e90bdf439c79d
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 158, in get_response
    % (callback.__module__, view_name))
ValueError: The view mysite.views.auth didn't return an HttpResponse object. It returned None instead.
2016-01-24 00:14:25,653 Not Found: /favicon.ico
2016-01-24 00:17:36,091 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 00:17:36,092 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 00:17:45,161 Not Found: /getauth
2016-01-24 00:17:53,760 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 00:18:19,120 Not Found: /favicon.ico
2016-01-24 00:18:35,647 管理员 进入 管理页面
2016-01-24 00:19:01,249 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-24 00:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 00:19:01,250 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-24 00:30:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 00:19:10,884 Internal Server Error: /auth/10011/22d43e99da90de72839399d98f5d3946
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 158, in get_response
    % (callback.__module__, view_name))
ValueError: The view mysite.views.auth didn't return an HttpResponse object. It returned None instead.
2016-01-24 00:19:11,126 Not Found: /favicon.ico
2016-01-24 00:19:40,056 Internal Server Error: /auth/10012/8a6983ca839a35e607575cbc8b01a95c
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 158, in get_response
    % (callback.__module__, view_name))
ValueError: The view mysite.views.auth didn't return an HttpResponse object. It returned None instead.
2016-01-24 00:19:40,285 Not Found: /favicon.ico
2016-01-24 00:21:49,247 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 00:21:49,248 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 00:21:53,223 Internal Server Error: /auth/10013/e721329a85399d2b0aca7c7555e58186
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 158, in get_response
    % (callback.__module__, view_name))
ValueError: The view mysite.views.auth didn't return an HttpResponse object. It returned None instead.
2016-01-24 00:21:53,455 Not Found: /favicon.ico
2016-01-24 00:22:48,763 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 00:22:48,768 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 00:22:55,585 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 00:22:58,598 /usr/local/lib/python2.7/dist-packages/bootstrapform/templatetags/bootstrap.py:80: RemovedInDjango110Warning: render() must be called with a dict, not a Context.
  return template.render(context)

2016-01-24 00:29:33,944 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 00:29:33,945 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 00:29:37,203 Internal Server Error: /auth/10014/6e004819ff1901b6a6fa767848d9fe05
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 158, in get_response
    % (callback.__module__, view_name))
ValueError: The view mysite.views.auth didn't return an HttpResponse object. It returned None instead.
2016-01-24 00:29:43,898 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 00:30:23,556 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 00:30:23,560 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 00:30:25,713 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 00:30:31,466 管理员 进入 管理页面
2016-01-24 00:30:48,241 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-24 00:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 00:30:48,243 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-24 23:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 00:30:53,611 管理员 进入 管理页面
2016-01-24 00:31:09,564 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-24 00:22:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 00:31:09,565 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-24 00:44:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 00:31:16,432 管理员 进入 管理页面
2016-01-24 00:35:59,734 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 00:35:59,734 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 00:45:11,392 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 00:45:11,392 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 00:45:16,674 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 00:45:16,913 Not Found: /favicon.ico
2016-01-24 00:45:18,381 管理员 进入 管理页面
2016-01-24 00:45:20,764 /usr/local/lib/python2.7/dist-packages/bootstrapform/templatetags/bootstrap.py:80: RemovedInDjango110Warning: render() must be called with a dict, not a Context.
  return template.render(context)

2016-01-24 00:45:42,747 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 00:45:42,748 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 00:46:22,488 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 00:46:22,649 Not Found: /favicon.ico
2016-01-24 00:47:01,467 Not Found: /favicon.ico
2016-01-24 00:48:39,734 Not Found: /favicon.ico
2016-01-24 00:48:45,497 /usr/local/lib/python2.7/dist-packages/bootstrapform/templatetags/bootstrap.py:80: RemovedInDjango110Warning: render() must be called with a dict, not a Context.
  return template.render(context)

2016-01-24 00:51:01,797 管理员 进入 管理页面
2016-01-24 00:52:07,997 管理员 进入 管理页面
2016-01-24 00:55:37,866 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 00:55:37,872 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 00:55:39,460 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 00:55:39,473 /usr/local/lib/python2.7/dist-packages/bootstrapform/templatetags/bootstrap.py:80: RemovedInDjango110Warning: render() must be called with a dict, not a Context.
  return template.render(context)

2016-01-24 00:55:46,429 Internal Server Error: /regexl
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/root/poll/mysite/views.py", line 134, in regexl
    addVoter(worksheet.cell_value(i,0), worksheet.cell_value(i,1), worksheet.cell_value(i,2), worksheet.cell_value(i,3))
  File "/root/poll/mysite/views.py", line 103, in addVoter
    User(idsn = idsn, username = name, phone = phone, type = int(type)).save()
ValueError: invalid literal for int() with base 10: ''
2016-01-24 00:57:42,437 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 00:57:42,438 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 00:59:23,138 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 00:59:26,375 /usr/local/lib/python2.7/dist-packages/bootstrapform/templatetags/bootstrap.py:80: RemovedInDjango110Warning: render() must be called with a dict, not a Context.
  return template.render(context)

2016-01-24 01:00:13,486 管理员 进入 管理页面
2016-01-24 01:00:33,211 /usr/local/lib/python2.7/dist-packages/django/forms/fields.py:849: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  if value == k or text_value == force_text(k):

2016-01-24 01:00:34,942 管理员 进入 管理页面
2016-01-24 01:00:41,524 管理员 进入 管理页面
2016-01-24 01:00:50,941 管理员 进入 管理页面
2016-01-24 01:01:04,014 管理员 进入 管理页面
2016-01-24 01:01:13,880 管理员 进入 管理页面
2016-01-24 01:01:28,308 管理员 进入 管理页面
2016-01-24 01:01:44,531 管理员 进入 管理页面
2016-01-24 01:02:04,376 管理员 进入 管理页面
2016-01-24 01:05:16,441 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 01:05:16,441 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 01:05:20,992 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 01:05:21,006 /usr/local/lib/python2.7/dist-packages/bootstrapform/templatetags/bootstrap.py:80: RemovedInDjango110Warning: render() must be called with a dict, not a Context.
  return template.render(context)

2016-01-24 01:08:32,078 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-24 01:09:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 01:08:32,079 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-24 01:15:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 01:13:47,028 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 01:13:47,028 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 01:13:49,264 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 01:14:44,634 Internal Server Error: /pollvote
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/root/poll/mysite/views.py", line 434, in pollvote
    curChoice = getChoice2(topic, getUser(request.POST["checkbox" + str(i)]))
  File "/root/poll/mysite/views.py", line 403, in getChoice2
    newChoice = Choice2(question = question, text = text)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/base.py", line 431, in __init__
    setattr(self, field.name, rel_obj)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/fields/related_descriptors.py", line 199, in __set__
    (instance._meta.object_name, self.field.name)
ValueError: Cannot assign None: "Choice2.text" does not allow null values.
2016-01-24 01:18:24,709 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 01:18:24,715 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 01:18:30,566 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 01:23:04,199 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 01:23:04,204 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 01:23:05,672 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 01:23:14,631 管理员 进入 管理页面
2016-01-24 01:23:36,456 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-24 00:20:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 01:23:36,457 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-24 00:30:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 01:23:39,378 管理员 进入 管理页面
2016-01-24 01:23:43,886 管理员 进入 管理页面
2016-01-24 01:24:22,164 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-24 01:20:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 01:24:22,165 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-24 01:40:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 01:31:38,604 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 01:31:38,605 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 01:32:00,547 管理员 进入 管理页面
2016-01-24 01:32:00,555 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 01:33:28,709 管理员 进入 管理页面
2016-01-24 01:34:26,638 管理员 进入 管理页面
2016-01-24 01:34:34,785 /usr/local/lib/python2.7/dist-packages/bootstrapform/templatetags/bootstrap.py:80: RemovedInDjango110Warning: render() must be called with a dict, not a Context.
  return template.render(context)

2016-01-24 01:52:06,637 /root/poll/mysite/urls.py:43: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^judge$', views.judge, name='judge'),

2016-01-24 01:52:06,640 /root/poll/poll/urls.py:12: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^',include('mysite.urls',namespace='mysite')),

2016-01-24 01:52:13,181 /usr/local/lib/python2.7/dist-packages/django/shortcuts.py:45: RemovedInDjango110Warning: The context_instance argument of render_to_string is deprecated.
  using=using)

2016-01-24 01:52:38,653 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-24 01:51:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 01:52:38,654 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-24 00:59:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 01:52:50,440 管理员 进入 管理页面
2016-01-24 01:53:07,070 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.st received a naive datetime (2016-01-24 01:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 01:53:07,071 /usr/local/lib/python2.7/dist-packages/django/db/models/fields/__init__.py:1453: RuntimeWarning: DateTimeField Question.dt received a naive datetime (2016-01-24 23:00:00) while time zone support is active.
  RuntimeWarning)

2016-01-24 01:54:30,949 管理员 进入 管理页面
